code-challenges.js

```diff
@@ -3,26 +3,49 @@
 
 // --------------------1) Create a function that takes in an array of objects and returns a sentence about each person with their names capitalized.
 
-var people = [{name: "ford prefect", occupation: "hitchhiker"}, {name: "zaphod beeblebrox", occupation: "president of the galaxy,"}, {name: "arthur dent", occupation: "radio employee"}]
+var people = [{name: "ford prefect", occupation: "hitchhiker"}, {name: "zaphod beeblebrox", occupation: "president of the galaxy"}, {name: "arthur dent", occupation: "radio employee"}]
 // Expected output example: "Ford Prefect is a hitchhiker." "Zaphod Beeblebrox is the president of the galaxy." "Arthus Dent is a radio employee."
 
 
+const personOccupation = (people) =>{
+    return people.map(function(item){
+        var action = item.occupation.includes("president")? " is the " : " is a "
+        var name = nameUpper(item.name)
+        return name + action + item.occupation + "."
+    })
+}
+const nameUpper = (name) =>{
+    return name.split(" ").map((item) => item.charAt(0).toUpperCase() + item.slice(1)).join(" ")
+}   
```

Comment:
i really love that you created the helper method `nameUpper` here. good thinking